docs/Users/FAQ.rst

```diff
@@ -0,0 +1,79 @@
+This is a list of frequently asked questions, aiming to answer any possible
+questions by newcomers or even contributors.
+
+Why did you choose the name?
+============================
+
+coala stands for "COde AnaLysis Application", works well with animals and thus
+is well visualizable, it's easy to memorize.
+
+Is there corporate backing behind coala? What are your intentions?
+==================================================================
+
+coala was founded for fun. coala was, is and will always be free software and is
+developed mostly by students and there's no corporate interest and no CLA. If
+you want to back us, contact us on our `gitter channel <https://gitter.im/coala-analyzer/coala>`__
+.
+
+What sort of analysis does coala do? What languages are supported?
+==================================================================
+
+A list of all analysis routines and supported languages is `here <https://github.com/coala-analyzer/bear-docs/blob/master/README.rst#supported-languages>`__
+- fully browsable.
+
+For a top level view on what languages support what kind of analysis roughly,
+consult `this link <https://docs.google.com/spreadsheets/d/1bm63TQHndmGf3HQ33fp9UEmGKNYI7dTkjMyFIof2PqA/edit?usp=sharing>`__
+.
+
+How do I get started with coala?
+================================
+
+If you're looking to get started using coala, we have a full tutorial `here <http://coala.readthedocs.io/en/latest/Users/Tutorials/Tutorial.html>`__
+that will teach you everything you need to know to use coala.
+
+If you're willing to contribute and become a part of our coalaian community,
+we have written a guide that will help you fix an issue on your own, just by
+following and understanding the indications `here <http://coala.readthedocs.io/en/latest/Getting_Involved/Newcomers.html>`__.
+It is meant for newcomers, and it does not require you to have any precedent
+knowledge regarding coala.
+
+How do I get in touch with you guys?
+====================================
+
+We are very active on our `gitter channel <https://gitter.im/coala-analyzer/coala>`__
+and will respond to any question in a matter of minutes. However, for a full
+list of how to get in touch with us, consult `this link <http://coala.readthedocs.io/en/latest/Getting_Involved/README.html>`__.
+
+What are those things failing/passing on my Pull Request?
+=========================================================
+
+We use a few checks to make sure your Pull Request is ready to be merged into
+our master branch. Right now we use 7 of those checks:
+
+- **review/gitmate/commit** Checks whether your commit message is alright and
+  doesn't have too many characters per line.
+
+- **review/gitmate/pr** Checks whether your code respects our styling (PEP8),
+  doesn't contain unneeded newlines, trailing whitespace, etc. Basically it is
+  like running coala over your code, so to fix this, simply run ``$ coala``
+  before pushing!
```

Comment:
the gitmate PR state will fail if any of the commit states fail so if the last comit is good but the previous ones aren't we want the PR to still have a failed state